### ISSUE TITLE
Unhandled pipeline class: DreamTexturesDepth2ImgPipeline

### ISSUE THREAD

**Issue by WilliamKappler** [carson-katri/dream-textures#738]
### Description

I'm using Blender 3.6.5 (also tried 3.6.2) on Windows 10 with a RTX 3060 12gb which can run Stable Diffusion via Automatic1111 and other programs without error. Dream Textures is version 0.3.1.

I set the addon up according to the instructions (using the manual approach) with the exception of attempting to use the two models I already have, SD1.4 and NAI. When I run the texture projection, I get the error:
```
bpy.ops.shade.dream_texture_project()
An error occurred while generating. Check the issues tab on GitHub to see if this has been reported before:
ValueError('Unhandled pipeline class: DreamTexturesDepth2ImgPipeline')
```

Maybe there's some connection to the models, since I'm not using the SD2.1 specified in the setup instructions. "Import Checkpoint File" in the options didn't work, but "Link Checkpoint File" did. Import did nothing and didn't report an error, but checking the python console I found:
```
[root removed for privacy]AppData\Roaming\Python\Python310\site-packages\torchvision\io\image.py:13: UserWarning: Failed to load image Python extension: [WinError 127] The specified procedure could not be found
  warn(f"Failed to load image Python extension: {e}")
```
Using the Link option added the model to the model list without complaint.

### Steps to Reproduce

Install as specified (manually unzipping to addons), link SD1.4 or NAI, create an icosphere and attempt to texture it with any prompt.
![dreamsetup](https://github.com/carson-katri/dream-textures/assets/13572055/eb9cb929-5a7a-49b1-9e7f-4037e0520209)

### Expected Behavior

The icosphere to be textured to at least some degree without error.

### Addon Version

Windows (CUDA)

### GPU

NVIDIA

**Comment by WilliamKappler** [comment]
Addendum: I messed around with this more and ended up prompted when I tried to do the projection that the model I have isn't suitable for using depth and I need a depth to image one. So, this seems like intended behavior and user-error on my part, _but_ perhaps then the real issue is that this error should trigger in what I originally did to avoid confusion.

I think the difference was in changing the model type during the linking to v1 instead of auto.
